dbt/models/sec10k/core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities/schema.yml

```diff
@@ -10,7 +10,7 @@ sources:
               config:
                 enabled: "{{ target.name == 'etl-full'}}"
               arguments:
-                min_value: 5544
+                min_value: 5543
```

Comment:
I guess this changed as a result of the EIA860 updates? It is an association table that links to EIA utilities.